app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark:int, brand:str):
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center:float, clean_power:int, average_rating:float, count_of_ratings:int):
```

Comment:
Alright, in this code we had small problem with flake8, ckeck problems with command in your terminal "flake8 app" and solve the problem.

Reply:
Fixed this problems now in new commit